### ISSUE TITLE
Testing feedback: general comments

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#1550]
As the tests come in for Gutenberg I am going to make issues but also add feedback here as a constant thread for people to check.

For reference the tests are here: https://make.wordpress.org/test/2017/06/27/call-for-testing-gutenberg/.

**Comment by karmatosed** [comment]
"I understand this is beta but it seems very limited if I can't even end a list and start another paragraph. It looks cool though and I think it'll be great when it gets more usable for real writing"

**Comment by karmatosed** [comment]
"A front-end version of Gutenberg would be awesome. Thanks!"

**Comment by karmatosed** [comment]
"The overall appearance is nice, but it feels like you're being spoon fed when trying to create content. This is mainly to do with the fact that once you press enter, a new content block that says "write..." always shows up. The UI seems to try and do too much and clutter the options altogether."

**Comment by karmatosed** [comment]
"I want to be a fan, but this has WAY TOO MANY ROUGH EDGES. Notable issues:
1) Clicking in the 'write' box at bottom generates extra blank blocks.
2) The "write' box at bottom never goes away, always feels like post is incomplete.
3) Buttons that show up only when you mouseover is a dangerous UI pattern. Think about explaining to your technically-challenged parent about clicking a button that doesn't exist for them until they put the mouse in just the right spot.
4) Attempts to embed Youtube URL via block failed.
5) Do you hate people who want to only use the keyboard?"

**Comment by karmatosed** [comment]
"I found I had to use the Trash icon to delete several blocks I added unintentionally. I found the Write placeholder initially helpful before I added my first block. After I'd added a block I found it unhelpful & actually confusing. I wasn't sure where clicking the + icon would add the new block."

**Comment by karmatosed** [comment]
"I really like the new editor. I look forward to using it. I hope that there a lots of hooks available, in order to customise it, if customer request extra features."

**Comment by karmatosed** [comment]
"For the most part I like the look and feel of the new editor, but I find it extremely awkward to write in. I have to leave the keyboard for the mouse a LOT to write more than few sentences."

**Comment by karmatosed** [comment]
"For the most part I like the look and feel of the new editor, but I find it extremely awkward to write in. I have to leave the keyboard for the mouse a LOT to write more than few sentences."

**Comment by karmatosed** [comment]
"Really not much. I dislike very much. 

I didn't feel it was intuitive to use. Breaking content into blocks such as this doesn't flow for me when writing a post. I also think that if people want to use blocks, there are plenty of page builders that approach content in this manner."

**Comment by karmatosed** [comment]
"Looks very good, feels modern. The option for full width images looks very nice!"

**Comment by karmatosed** [comment]
"The decluttering of the interface and the block system. They improved a lot the experience. Great work so far!"

**Comment by karmatosed** [comment]
"I don't see it as an approach to writing that yields itself to business use of WordPress. Particularly the current bug in copy pasting. Many clients hand me copy to paste into their pages. This issue must be fixed before this is ready for prime time.

What steps will you take to prevent sites from "breaking" when this is introduced. It seems like a huge undertaking for all programs and plugins to come into compliance that are currently built on Tiny MCE editor."

**Comment by karmatosed** [comment]
"I somehow expected more WYSIWYG features, e.g. font selection, font size etc."

**Comment by karmatosed** [comment]
"I found that adding text to the button block wasn't very intuitive. 

Sure, after a while I realize that you can just start typing to add text, but it's not obvious, because the placeholder text ("Paste URL or type") seems related only to the link itself – not the button text. 

I was expecting some visual feedback when I clicked on the button to add the text, when I finished inserting the URL."

**Comment by karmatosed** [comment]
"I think this has potential but you need to get it into the hands of people who write long form content. 

Writing in Gutenberg needs to be like a bobsled run, whereas right now it is running a steeplechase, you never know what the next obstacle will be that may cause you to trip and fall on your face.

TinyMCE has it's own hurdles as well, like the damned line break and p tag mysteries, but at least I could just write smoothly and come back to clean up the formatting later (or at least try). With Gutenberg I can't even get smoothly to the next paragraph."

**Comment by karmatosed** [comment]
"Whatever happened to keeping WordPress basically "vanilla" and letting plugins do the customizing? As, mentioned before if I wanted to produce content like this I would use a page builder. Hope that you make this a choice rather than the final option for adding content."

**Comment by karmatosed** [comment]
"The current editor is very pluggable, and can be replaced by e.g. page builder plugins. Hope this will remain with Gutenberg! Since: Gutenberg seems to be focused very much on content writing. This makes sense and is perfectly OK. But it does not replace e.g. page builders, which serve somehow different purposes (building complete website layouts w/o coding)."

**Comment by karmatosed** [comment]
"Please, please, keep the automatic render of the embed, just like is done in the current TinyMCE editor... 

In the new editor, you need five actions to add a new video: click "Insert" + scroll to find the embed + click the embed + click the URL filed + paste the link + click "Embed". 

In the current editor, you need just two actions: click the editor window and paste the link. Imagine the economy of clicks when inserting lots of embeds in a page, guys. 

Also, IMHO there's no need to put all the possible options of embeds upfront in a section called "Embed", inside the "Insert" menu. 

Just a tiny fraction will be used anyway by pretty much everyone. Maybe just adding a "Embed" option to show all the options after that, or even automatically adding the option in the "Embed" section only after was used a couple of times, I don't know. 

Well, at any rate, thanks a lot for making WordPress cool again! :)"

**Comment by jwold** [comment]
Check other tickets since I slept

> write placeholder

1. Several pieces of feedback suggested getting rid of the word “Write”. What if we had it disappear if there are more than 3 blocks present?

> “Particularly the current bug in copy pasting. Many clients hand me copy to paste into their pages. This issue must be fixed before this is ready for prime time.”

2. I wonder what exactly they were saying is the problem here?

> "I think this has potential but you need to get it into the hands of people who write long form content.

3. Great feedback! I’ve started using it in the past day to write articles. I think as we get more feedback from people creating articles with it regularly we’ll be able to make tweaks to improve it. Already found a few things just from the first two articles I wrote. 

> "Whatever happened to keeping WordPress basically "vanilla" and letting plugins do the customizing? As, mentioned before if I wanted to produce content like this I would use a page builder. Hope that you make this a choice rather than the final option for adding content."

4. WordPress can continue to allow for more customization but there can be a line between what it offers and what plugins offer (i.e page builders). Finding that middle ground will be a great problem to solve, but I believe it’s there somewhere. With that said I personally don’t see an issue with the direction we’re taking Gutenberg. I believe it’s a great improvement and, if we can make tweaks to allow for long form content entry to flow, then we may solve some of these concerns about changing up the editor. 

> "The current editor is very pluggable, and can be replaced by e.g. page builder plugins. Hope this will remain with Gutenberg! Since: Gutenberg seems to be focused very much on content writing. This makes sense and is perfectly OK. But it does not replace e.g. page builders, which serve somehow different purposes (building complete website layouts w/o coding)."

5. Ultimately I think this will be ok if page builders can plug into the new editor as well. 

> Also, IMHO there's no need to put all the possible options of embeds upfront in a section called "Embed", inside the "Insert" menu.

Lots of discussion has been had on this point. I think the resolution for the moment is to start by showing all and we can always backtrack later if need be.

**Comment by karmatosed** [comment]
"I found the editer in its standard form generally fine to use.
I don't however need all the extra guff for a normal wordpress post, in fact I could care less about all that stuff.
The button panel is clunky, and well, some things just didn't work at all."

**Comment by karmatosed** [comment]
"Please leave the gutenberg editor as an option disabled by default.
All the site that i made use an advanced editor"

**Comment by karmatosed** [comment]
"This editor is very useful and flexible within it's block style but I have faced problem for aligning the blocks makes clicking into content areas is bit difficult for me."

**Comment by karmatosed** [comment]
"No preview, Can’t save a draft and preview, Can’t edit the code of the page/post
Can’t add a category editing the post"

**Comment by karmatosed** [comment]
"I'm concerned as a plugin developer how meta boxes for data that requires more space than a simple sidebar, i.e. meta boxes that used to go below the editor."

**Comment by karmatosed** [comment]
"Very intuitive to use. I expected the contextual menus to be extremely annoying, but they are "gentle" and not too intrusive"

**Comment by karmatosed** [comment]
"Looking at the rendered HTML, I'm surprised to see all paragraphs currently wrapped in double-quotes."

**Comment by karmatosed** [comment]
"Too much clicks. Not practical. Confusing. Decreases customization. Buggy."

**Comment by Ipstenu** [comment]
I'm not sure where you want blog-post reviews posted but ... - https://halfelf.org/2017/post-written-gutenberg/

The part that's in markdown obviously didn't get converted, though it did in the post previews. So that's interesting.

**Comment by karmatosed** [comment]
"For this specific test it was easy to use. The test did not address things like columns though. From the tinkering I've done a lot of work is still needed there. The image block and embeds seem to have some sizing and alignment options but they are very glitchy for me and do not offer enough control. What if I want two image blocks each taking 1/4 of the screen and an additional text block taking the remaining 1/2. There are plenty of solutions for achieving this kind of thing already with plugins like ACF. Obviously you all know this but without that kind of control this will never work for me. I'm also concerned about what happens to the current custom solutions I and may others have built to handle these sort of scenarios. If this new experience is forced and kills those features I will potentially have several hundred broken sites on my hands. It will probably be the end of WordPress me. I love WordPress. I don't want that to happen. "

**Comment by karmatosed** [comment]
""The UX and UI still need a lot of work. Most of my clients would need a lot of hand-holding to find the things they use frequently. Even the Plus sign in the sticky ribbon isn't obvious to find all the options because the person creating the post will be so focused on the plus sign on the page itself. Having the ability to add blocks directly inside the visual editor immediately retrains the brain to longer pay attention to the ribbon. 

The biggest improvement would be to have all the options for blocks appear on Hover instead of requiring a click.""

**Comment by karmatosed** [comment]
"I tried the Embeds feature. Nothing worked there. Tried Funny or Die and College Humor. Copied and pasted URL into the box and hit Embed. Got back "Sorry, we could not embed that content." Tried the URL and tried Embed code. Same result. "

**Comment by karmatosed** [comment]
"I tested this editor with VoiceOver. It was very difficult to navigate between blocks and options for formatting. If I have more than 2 blocks moving from one block to the other is a long way. I need to pass through each menu and button associated to the block. It is very tiring. Also, assessing options in the menus don't respond well to VoiceOver commands. I was often redirected to other places in the page like ""post settings"" or ""skip to main content"" link.
When editing text in a block, it is better to select an option in a menu before writing content. If text is already in the block, it is not possible to select it and then select an option in the menu (like for example select a list, or bold or italic).
The insert button it was not intuitive, it took me a while before finding how to insert a block image. 
"

**Comment by karmatosed** [comment]
"There UI is too "slick" and, without texts for the icons it is often very unclear as to what they do. Also, important things such as changing the post type are currently not supported - which a lot of us need!"

**Comment by karmatosed** [comment]
"Overall, a very positive user experience! I do feel there could be some slight refinement in how a few of the options are presented and possibly some guided assistance in the form of tooltips or small text near some of the icons for those less familiar with editing content on the web."

**Comment by karmatosed** [comment]
"I really like the new editor experience, it's nice to have a more graphical / visually appealing experience when creating a page, though I do feel there could be some added hints or tooltips to make it clear what options are available.  However, I do think that for most people with experience using a modern word editor or website builder, it is fairly intuitive!"

**Comment by karmatosed** [comment]
"Overall I do like it. I do feel there might still be a few UI / UX things to work on. Eg. I find it weird to go to the left of the screen every time I need to save. It should be closer and easier to see. Maybe more prominent than Publish? The delete/bin icon is also too close to the settings icon next to each block. I accidentally deleted a block, instead of going in settings. What happens if this happens. Is there an undo?"

**Comment by karmatosed** [comment]
"The preview is not available, trying something new I would like to see how it looks before posting. Overall it is easier to write and post with the current WP of doing things"

**Comment by karmatosed** [comment]
""In it's current state, it feels super rough and unusable.  It feels like having a bunch of blocks and not a ""post"" any more.  I like the feeling of things being one post instead of a bunch of separate parts.  I also ended up with a bunch of empty blocks because I wasn't able to move them easily and I needed to create different ones.

Overall, images, bullet points, and quotes were already really easy in WordPress, this actually makes them a bit more complicated.  If this gave me additional features that are NOT easy right now (layouts, columns, extra options, wrapping text around images, blocks within blocks) I would consider switching over, but this doesn't add any user candy right now.""

**Comment by karmatosed** [comment]
"The editor right now is only good for single column layout. When I tried on a different test to right align an image, it didn't work and I had no way to re-size it. Also, there really needs to be a column for regular content option. This is one of my most frequent client requests.
Do keep up the good work!"

**Comment by karmatosed** [comment]
"The text editor tools get the way of the text just written. 

After creating a second paragraph it's not possible to arrow the cursor back up to the first paragraph. I'm ""locked"" into a tiny edit window.

Editing a post in ""text"" view is very error-prone. It's much harder to scan and view actual content amidst all the new HTML comments. Accidentally deleting or editing a comment deletes actual content when switching to visual mode.

In general, it's much harder to get in the flow of writing long-form prose in tiny blocks. There are too many distractions in either Visual or Text mode."

**Comment by karmatosed** [comment]
"I added a list, then a quote and then a paragraph with a drop cap to see how everything would stack up.  Looks great on the backend. Looks terrible on the front end - the quote and paragraph with drop cap are crashing into each other. I think it would be best to have a default padding set for above and below a quote to give it some air, though I know we don't want to trend too much into appearance, as that should be handled mostly by the theme.  (fyi: I had Twenty Seventeen as the default theme while testing)."

**Comment by karmatosed** [comment]
"Showed it to some of my business clients, nobody liked it. Editing is more complicated, takes more mouse actions, navigating with keyboard impossible, bad copy and past experience (e.g. having to split blocks to format headings, list, insert pics at random places with custom sizes and and and). 

Does not address column layouts or formatting needed to create to result wanted, page builder still needed even for simple pages.

Kills custom styles, color, shortcode buttons. (Custom shortcodes are a very simple way of extending the editor which can be handled with almost no effort, don't force customers to build their own blocks with a complex api) 

Custom meta boxes (including multiple editors or no editor at all) are crucial. No way to go without it. (And don't call it ""additional settings"", this IS the content)

UI with buttons popping up and disappering at multiple places, sometimes covering the text you want to edit, is distracting and creates a nervous and restless experience. 
"

**Comment by karmatosed** [comment]
"Breaking backwards compatibility in such a way means killing sites, investments, peoples time, skills, productivity.

Therefore lots of people will never update, what does that mean for security?

Never make that thing mandatory. Maybe make it a plugin for bloggers."

**Comment by karmatosed** [comment]
"I think Gutenberg might be an excellent option for certain types of writers or certain types of content. I'm very worried that once the Gutenberg plugin moves into core we won't have an option and will be forced to use this "block"-type editor for everything."

**Comment by StaggerLeee** [comment]
Explain to people better what Gutenberg is, will be. Seems as you are holding for yourself final destination of Gutenberg. Share it without backthoughts what it will be, or you imagine it as, when it goes final.

There are many negative Articles, and many negative reviews in plugin repository.

Separate it to 4 levels (very detailed):

- Explain to very beginners, those who can barely make strong text in MS Word, what benefits will be for them (mostly website owners and editors self).
- Explain to plugin developers what benefits will be for them.
- Explain to theme developers what benefits will be for them.
- Explain to website designers, developers what benefits will be for them (those who dont make themes/plugins, but make/do/finish website as final product).

**Comment by karmatosed** [comment]
"There needs to be an awareness of the difference between a general user, who can do whatever they like and a corporate user who needs to locked into a restriction of styles. For instance, I might allow h3  and h4 but not h1 and h2. I might allow 2 columns but not more.  I will assume that there will be a mechanism for front-end styles to be used. Custom fields?"

**Comment by rianrietveld** [comment]
@karmatosed, I've got 2 test remarks back in my email on the general test (done on version 0.8.0):

One of a blind web developer that gave up on the test (with Jaws and IE):
> I couldn’t for the life of me figure out how to add and edit the image and when I took the survey it crashed my browser.

One of Eric Wright, who is an accessibility expert:
```
Guided Gutenberg task complete. I forgot to include it on my forms, but I used Dragon Professional Individual 15 and Chrome 60 stable.
My biggest barrier right now is that I can't use any of the controls that appear when you mouse-over a block. I feel like core functions (delete, move up, move down) are hidden. I can only get them to appear briefly, and I can't activate them when they do appear. 
```

**Comment by annaephox** [comment]
Analysis/[summary of feedback in this ticket to date here](https://go.tinymce.com/blog/gutenberg-editor-success-lies-timing/)

**Comment by karmatosed** [comment]
"Since I want to create media rich long-form posts I really appreciate the Blocks concept an a very close to the final result WSIWYG experience. Especially the easy reordering makes it easy to create individual layouts. I like the clean layout of the editor and straightforward editing which makes WP competitive again against the blog platform like medium, exposure, ghost etc"

**Comment by karmatosed** [comment]
" be cool if it would auto-format to the correct citation hyphen. 

After I paste in my post, I wanted to add an image to the middle of the text but couldn't figure out how: http://cld.wthms.co/cBivpm

I expected to see an option there to make it an image. It wasn't until I scrolled all the way down that I saw how you add an image. Any reason we wouldn't want to add an image option to the content block? It may be that we don't want to keep adding to that menu, but then that makes me wonder how we will add other content types. 

I wanted to be able to make a box style and add a button but couldn't. 

I didn't know what preformatted meant. Once I selected it and it changed the text style I thought maybe that meant I could treat that content block like the text editor in wp-admin now, so I tried my [box] style that way. Once I published I realized what that meant, but styling was messed up. 

I also don't know what verse would mean and it messed up styling in the editor: http://cld.wthms.co/0NyJlF

It wasn't immediately apparent to me how to save as a draft since I was looking by publish. Seemed odd to me to have Saved on the other side of the page: http://cld.wthms.co/rqvTIC"

**Comment by karmatosed** [comment]
"Well when hitting the buttons there was no feadback they were ever pressed.  There was also a lot of html type code all through the editer content field, I don't know what that was but I want it gone.
I also couldn't tell when creating lists, incerting images, etc, maybe those should be in its own fields maybe select how many items in list from the drop down box once you hit that button, for say ol or ul bulleted or just numbered, etc, and how many items then type them on one line maybe with a , after each one or on seperate lines and just well they go in when you save or publish the document.
And thats the other thing no one knows or cares what ul or ol or all the other buttons are, they should be spoken in full what they are, ie ol is a numbered list, ul is a bulleted list unless you are an html programmer you wouldn't know what it was.
apart from that the editer is better, the older one is still better though."

**Comment by karmatosed** [comment]
"I'm sure you know that it doesn't place images well. I also just noticed that it doesn't give a list of most used tags. I write for a lot of blogs and I appreciate that list so I don't have multiple similar tags because I forgot how I phrased it last time."

**Comment by karmatosed** [comment]
"I didn't have any problems exactly, but I did think that it was strange that, depending on which option I had for a text block, the options it gave me when I clicked for more would change. For example when I added a Header, then clicked the H, my options are Paragraph or Quote. If I click Quote, my options are List, Paragraph, and Header."

**Comment by karmatosed** [comment]
"Was quite confusing to get started with given some expectations I had from other editors

In terms of actually testing it out one confusing thing though maybe only something for technical users was that I tried to install a release from github and forgot that the releases there are only zips of the source code and require another build step to use. The failure mode for this was that the plugin installed fine but gave a blank page with no error message and only showed it was 404ing on the assets being missing when I opened the chrome dev tools. The call out on the repository to install not from the releases list but from the wordpress plugin repository was invisible to me."

**Comment by karmatosed** [comment]
"I initially expected block drag-and-drop - although once I became comfortable with the arrow system that was immediately fine. 
I couldn't find the image formatting options for wide and full-width. I eventually realised it was a theme issue (Twenty Twelve, Baskerville 2, Toujours). After activating the Gutenberg theme from the Github repo I found the formatting options I was looking for."

**Comment by karmatosed** [comment]
"Please take notice of the following:
- if I right-align the button block, it wraps around the block that comes below (ie latest post block) instead of being independent.
- the image caption get centered regardless of the image position.
- I can't centre align an image.
- there is no "no align" option for an image.
- the column align centre feature for text doesn't work
- the button block doesn't allow me to add text to the button
- the button centre align feature doesn't work
- despite I added a url to the button link, it didn't work in the front end
- I'm not able to add a paragraph block under a right-aligned image without wrapping the paragraph to the left of that image.
- there is no much padding between some blocks"

**Comment by karmatosed** [comment]
"Changing the alignment of images has no effect on the post. In edit it shows right or left aligned; however, it is posted center. Images posted with regular editor shows right aligned image. Image may need to be resized for alignment to appear, but there is no indication of that from the Gutenberg editor. Also when posting one image after another in Gutenberg, the editor shows them as overlapping or places the caption for the bottom one next to the upper one. The same thing happened when trying to right align and image and add a left side pull quote. The quote ended up being below the picture."

**Comment by karmatosed** [comment]
"I didn't know how to float an image next to a paragraph. I had to ask. "

**Comment by karmatosed** [comment]
"- it'd be nice to have custom column layouts such as 1/3 + 2/3, ...
- it'd be nice to see the permalink like in the current editor.
- will you add an icon block?
- can a move a group of blocks as opposed to only one?
- it'd be nice to have a block duplicating feature.
- can I make a button block open the link in a different tab/window?
- the column layout is not responsive, ie a 4 columns on desktop will stay the same on mobile.
- will you add a tiles block? or grid block?
- re image gallery:
-- I wish I could use the arrows to navigate from one image to the other once I clicked on them.
-- I wish there was an X to return to the post OR could use the ESC key to achieve the same.
- it'd be nice to be reminded what kind of block I'm dealing with if I'm going back to one after sometime, I might not remember what it was, they can be confused between all the text blocks (paragraph, classic text, cover text, verse, ...)
- it'd be nice to see the tooltips back over the classic text tmce toolbar block"

**Comment by karmatosed** [comment]
"Inconsistency and not being clear in quote section made me a bit hesitant to use this editor as my main post editor"

**Comment by karmatosed** [comment]
"I like that it is clean design but I believe the move to icons is a bit confusing for regular users and some of the icons are not very descriptive and easy to find"

**Comment by karmatosed** [comment]
"The Block settings that appear on the right are hard to spot, especially on a large monitor. Maybe focusing on that area after the gear icon is clicked would be worth exploring? Also, finding Save button (link?) took me a while, I expected it to be somewhere near the Publish button.
Keep up the amazing work!"

**Comment by karmatosed** [comment]
"When ranging the image left, for instance, the next block goes right as expected but I if it doesn't reach vertically past the image, I would need a 'clear' before the subsequent block "

**Comment by karmatosed** [comment]
"After publishing I wanted to see the post. The preview button is on the right but the view post button is on the left and it's a small link. It would make sense to me if they are in the same area."

**Comment by karmatosed** [comment]
"There is a lot of getting used to, I can imagine for new users as well. I do like the idea of blocks, and the 'block creation' after for instance an enter is awesome and quick (local install).

What I am really missing in the plugin is a way to use Gutenberg for posts that are already created. When you hit publish, or accidentally are thrown out of the editor, there is no easy way (is there any?) to get back to editing in the GB editor."

**Comment by karmatosed** [comment]
"When I pressed the Preview button first time "about:blank" page opened in the new tab and nothing was there. I pressed it a couple more times but nothing changed. I had to update the post and then open the post (via permalink) to view the changes. When I tried to press Preview again after that, it worked.

Another issue I noticed, is that I updated the post and then left clicked the permalink to view the post in the same browser tab, but when I pressed browser back button to go back, the editor opened with empty content, like it was a new post. After I refreshed the editor page the content was back."

**Comment by karmatosed** [comment]
"In some ways, I really like the design of the new editor and would find a pleasant place to type longer articles/posts. One of the downsides I see is that by splitting everything up into blocks, it seems to take longer to make a post that has multiple fields/types. Also I wonder, having not yet tried, how easy/difficult it would be to add an image that would float to the left/right of a paragraph. 

Additionally, there does appear to be bugs with some block types that make the floating bar editor impossible to use. (One is detailed above. In trying to left/right float an image in "

**Comment by karmatosed** [comment]
"It's not intuitively clear how to write a post fluidly. Having the different elements, paragraph, lists, quotes in different boxes breaks up the flow of the page. Typically when I write I input all the text and then go back a highlight specific text to change to a list or a quote, for instance. I also typically add photos after the text is there, adding it beforehand is confusing. This is my first time trying it out so I may be missing something. Thanks."

**Comment by paaljoachim** [comment]
It would be good to take a closer look at the feedback received here:
https://yoast.com/dev-blog/user-testing-gutenberg/

**Comment by karmatosed** [comment]
@paaljoachim I have already asked Tim to make issues.

**Comment by karmatosed** [comment]
Closing as moving to the project. We are moving things to projects that aren't core Gutenberg and need to be done apart.
